New "light" footer color scheme - Closes #30

kaelig opened the pull request:
Following remarks from both the Interactive Graphics team and Marketing, for whom the dark color of the footer was too strong and bold, here is a lighter-looking version.

![screenshot 2015-07-29 16 59 12](https://cloud.githubusercontent.com/assets/85783/8962654/a071ac52-3613-11e5-9d34-5a5e5482752f.png)

/cc @kavanagh @danskinner

kaelig commented:
@kavanagh would that be a version of the footer you'd use?

danskinner commented:
Looks good in general. Only point I'd make is that all text is black, links and titles. We only use black as a link colour for article headlines and module titles - so we could potentially use it but would still need the titles to stand out, with bold perhaps.

Might be better to drop the links back to #777. We don't generally use this as a link colour either but do in at least one exception and it ought to be pretty obvious from the layout that they're links - would be more subtle too which fits with its use case.

kavanagh commented:
Yep, we'll use that.

Is this beta atm? I wonder what this would look like if you wanted only a simple footer: the copyright and Pearson strap (no site links). Would it still look ok?

kaelig commented:
@danskinner The default footer has all-white links and titles.

See screenshot:
![screenshot 2015-07-30 12 58 45](https://cloud.githubusercontent.com/assets/85783/8982568/bc3e66c6-36ba-11e5-894a-a575dbd97f1c.png)

We should probably keep the same pattern across both themes (or fix it in both). Also, `#777` fails WCAG AA contrast check on `#f6e9d8` (pink-tint1).

----

@kavanagh a beta is coming soon :)

Latest changes:

- removed the branding colour because it was coming as very strong
- Made the copyright text slightly bigger
- Removed space between the two parts of the copyright (fixes #28)
- Added a simpler version with no navigation

![screenshot 2015-07-30 17 02 29](https://cloud.githubusercontent.com/assets/85783/8988236/dd423d58-36dc-11e5-8f8c-4d1da20fbdde.png)

![screenshot 2015-07-30 17 02 23](https://cloud.githubusercontent.com/assets/85783/8988240/df825d32-36dc-11e5-84b1-58eb8c1f5836.png)

Does this work for you @danskinner @kavanagh?

danskinner commented:
Excellent point about the current/white design :) Though I'd say it's only like that because it was never fully iterated and finished. Definitely needs to be some distinction between the links/titles.

I suggest we just make the titles bold for now and use that across both versions, keeping the colours as they are.

Otherwise, all looks good to me.

kavanagh commented:
:+1:

kaelig commented:
Great, I made the titles bold, it works really well.

![screenshot 2015-07-31 13 54 01](https://cloud.githubusercontent.com/assets/85783/9007684/9c798114-378b-11e5-980a-81b73f8ae54f.png)
![screenshot 2015-07-31 13 54 19](https://cloud.githubusercontent.com/assets/85783/9007692/a8c5ae70-378b-11e5-93e4-b4adb18450ce.png)


![screenshot 2015-07-31 13 53 35](https://cloud.githubusercontent.com/assets/85783/9007677/8f3d403a-378b-11e5-9885-6d560a2ab171.png)
